src/main/frontend/handler/extract.cljs

```diff
@@ -0,0 +1,184 @@
+(ns frontend.handler.extract
+  "Extract helper."
+  (:require [frontend.util :as util]
+            [frontend.db :as db]
+            [lambdaisland.glogi :as log]
+            [clojure.set :as set]
+            [frontend.utf8 :as utf8]
+            [frontend.date :as date]
+            [clojure.string :as string]
+            [frontend.format.mldoc :as mldoc]
+            [frontend.format.block :as block]
+            [frontend.format :as format]
+            [cljs-time.core :as t]
+            [cljs-time.coerce :as tc]))
+
+(defn reset-contents-and-blocks!
```

Comment:
Why is this function in `extract.cljs`?

Reply:
Indeed, just moved it to `handler.repo`.